frontend/src/components/navigation/Header.css

```diff
@@ -0,0 +1,150 @@
+.NavbarItems {
```

Comment:
This is likely my bad for not clarifying this, but our project is going to use [Styled Components](https://styled-components.com/). This will allow us to better build components and fit more nicely with our React frontend, [among other benefits](https://blog.logrocket.com/8-reasons-to-use-styled-components-cf3788f0bb4d/).

The basic setup of the styled component for the header is in the file `frontend/src/components/navigation/Header.tsx`; you commented this out, I would recommend un-commenting this out and then working to transfer the work done here into the form of styled components. If you have any questions about this, please let @zpChris (me!) know 😃